taskcluster/docker/node/build.py

```diff
@@ -288,10 +288,10 @@ def main():
 
         dest = os.environ['XPI_ARTIFACTS']
         os.makedirs(os.path.dirname(dest), exist_ok=True)
-        shutil.copyfile(
-            f"{objdir}/dist/xpi-stage/{xpi_name}@mozilla.org.xpi",
-            dest
-        )
+        stage_xpi = glob.glob(f"{objdir}/dist/xpi-stage/{xpi_name}@*.xpi")
+        if len(stage_xpi) != 1:
+            raise Exception(f"Could not find {xpi_name}@*.xpi: {stage_xpi}")
```

Comment:
I'm assuming that we can't get more than 1 XPI as a result but if that ever happens, that error message will be very confusing

Reply:
Fair.  Something like "Expected exactly one {xpi_name}@*.xpi, but found {stage_xpi}" maybe?